pom.xml

```diff
@@ -1,426 +1,459 @@
-										<include>**/*.*</include>
-									</includes>
-								</resource>
-							</resources>
-						</configuration>
-					</execution>
-				</executions>
-			</plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-failsafe-plugin</artifactId>
+        <version>${maven-failsafe-plugin.version}</version>
+        <executions>
+          <execution>
+            <phase>integration-test</phase>
+            <goals>
+              <goal>integration-test</goal>
+              <goal>verify</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <properties>
+            <property>
+              <name>listener</name>
+              <value>org.sonar.java.jacoco.JUnitListener</value>
+            </property>
+          </properties>
+          <argLine>-Dfile.encoding=UTF-8</argLine>
+          <argLine>${failsafeArgLine}</argLine>
+          <classesDirectory>${project.build.outputDirectory}</classesDirectory>
+          <skipITs>${skip.integration.tests}</skipITs>
+          <includes>
+            <include>**/*ITest.java</include>
+          </includes>
+        </configuration>
+      </plugin>
 
-			<plugin>
-				<groupId>com.google.cloud.tools</groupId>
-				<artifactId>jib-maven-plugin</artifactId>
-				<version>${jib-maven-plugin.version}</version>
-				<configuration>
-					<to>
-						<image>169942020521.dkr.ecr.eu-west-2.amazonaws.com/local/officer-delta-processor:latest</image>
-					</to>
-				</configuration>
-			</plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-resources-plugin</artifactId>
+        <version>${maven-resources-plugin.version}</version>
+        <executions>
+          <execution>
+            <id>copy-resources</id>
+            <phase>process-resources</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${basedir}/target/test-classes</outputDirectory>
+              <resources>
+                <resource>
+                  <directory>${basedir}/src/itest/resources</directory>
+                  <includes>
+                    <include>**/*.*</include>
+                  </includes>
+                </resource>
+              </resources>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
 
-		</plugins>
-	</build>
+      <plugin>
+        <groupId>com.google.cloud.tools</groupId>
+        <artifactId>jib-maven-plugin</artifactId>
+        <version>${jib-maven-plugin.version}</version>
+        <configuration>
+          <from>
+            <image>
+              416670754337.dkr.ecr.eu-west-2.amazonaws.com/ci-corretto-build-21:latest
+            </image>
+          </from>
+          <to>
+            <image>
+              169942020521.dkr.ecr.eu-west-2.amazonaws.com/local/officer-delta-processor:latest
```

Comment:
do you want to just update this to shared services image now? think we confirmed that it doesn't cause an issue right?